### PR TITLE
Fix API test names according to new naming conventions (part 1)

### PR DESCRIPTION
**Pull request by abalakh**

### CODE REVIEW

File: tests/foreman/api/test_activationkey.py [hunk 1]
```diff
@@ -78,8 +78,8 @@ def test_positive_create_description(self):
                 act_key = entities.ActivationKey(description=desc).create()
                 self.assertEqual(desc, act_key.description)
 
-    @tier2
-    def test_negative_create_no_content_host_limit(self):
+    @tier1
+    def test_negative_create_no_chost_limit(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_activationkey.py]: ```test_negative_create_with_no_chost_limit```
or
```test_negative_create_without_chost_limit```

**Comment by elyezer** [comment]
ACK

File: tests/foreman/api/test_activationkey.py [hunk 2]
```diff
@@ -90,8 +90,8 @@ def test_negative_create_no_content_host_limit(self):
         with self.assertRaises(HTTPError):
             entities.ActivationKey(unlimited_content_hosts=False).create()
 
-    @tier2
-    def test_negative_create_invalid_content_host_limit(self):
+    @tier1
+    def test_negative_create_invalid_chost_limit(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_activationkey.py]: ```test_negative_create_with_invalid_chost_limit```

File: tests/foreman/api/test_activationkey.py [hunk 3]
```diff
@@ -107,9 +107,9 @@ def test_negative_create_invalid_content_host_limit(self):
                         unlimited_content_hosts=False,
                     ).create()
 
-    @tier2
+    @tier1
     @skip_if_bug_open('bugzilla', 1156555)
-    def test_negative_create_no_content_host_limit_with_set_max(self):
+    def test_negative_create_no_chost_limit_with_set_max(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_activationkey.py]: maybe something like `test_negative_create_without_chost_limit_and_set_max`

File: tests/foreman/api/test_contentview.py [hunk 4]
```diff
@@ -38,7 +39,7 @@ class ContentViewTestCase(APITestCase):
 
     @tier3
     @run_only_on('sat')
-    def test_subscribe_system_to_cv(self):
+    def test_positive_subscribe_system_to_cv(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_contentview.py]: not sure whether we need that `_to_cv`

File: tests/foreman/api/test_contentview.py [hunk 5]
```diff
@@ -173,7 +174,7 @@ def test_negative_associate_puppet_content(self):
 
     @tier2
     @run_only_on('sat')
-    def test_negative_associate_dupe_content(self):
+    def test_negative_add_dupe_content(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_contentview.py]: ```test_negative_add_dupe_repos```

File: tests/foreman/api/test_contentview.py [hunk 6]
```diff
@@ -1199,8 +1215,9 @@ def test_custom_cv_subscribe_system(self):
         # This test is implemented in tests/foreman/smoke/test_api_smoke.py.
         # See the end of method TestSmoke.test_smoke.
 
+    @tier2
     @stubbed()
-    def test_cv_dynflow_restart_promote(self):
+    def test_positive_dynflow_restart_promote(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_contentview.py]: To have same approach as in UI and basically action should be located on third place
`test_positive_restart_promote_via_dynflow`

File: tests/foreman/api/test_contentview.py [hunk 7]
```diff
@@ -1211,8 +1228,9 @@ def test_cv_dynflow_restart_promote(self):
         @status: Manual
         """
 
+    @tier2
     @stubbed()
-    def test_cv_dynflow_restart_publish(self):
+    def test_positive_dynflow_restart_publish(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_contentview.py]: To have same approach as in UI and basically action should be located on third place
`test_positive_restart_publish_via_dynflow`

File: tests/foreman/api/test_contentview.py [hunk 8]
```diff
@@ -1245,8 +1264,9 @@ def test_cv_roles_admin_user(self):
         # Variations:
         #  * Read, Modify, Delete, Promote Publish, Subscribe
 
+    @tier2
     @stubbed()
-    def test_cv_roles_readonly_user(self):
+    def test_positive_roles_readonly_user(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_contentview.py]: We have another approach with UI. Check whether you like to use it here

File: tests/foreman/api/test_contentview.py [hunk 9]
```diff
@@ -1265,8 +1285,9 @@ def test_cv_roles_readonly_user(self):
         # Variations:
         #  * Read, Modify,  Promote?, Publish?, Subscribe??
 
+    @tier2
     @stubbed()
-    def test_cv_roles_admin_user_negative(self):
+    def test_negative_roles_admin_user(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_contentview.py]: We have another approach with UI. Check whether you like to use it here

File: tests/foreman/api/test_contentview.py [hunk 10]
```diff
@@ -1284,8 +1305,9 @@ def test_cv_roles_admin_user_negative(self):
         # Variations:
         #  * Read, Modify, Delete, Promote Publish, Subscribe
 
+    @tier2
     @stubbed()
-    def test_cv_roles_readonly_user_negative(self):
+    def test_negative_roles_readonly_user(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_contentview.py]: We have another approach with UI. Check whether you like to use it here

File: tests/foreman/api/test_contentviewfilter.py [hunk 11]
```diff
@@ -220,7 +219,7 @@ def test_positive_create_with_original_packages(self):
 
     @tier2
     @run_only_on('sat')
-    def test_positive_create_with_multiple_repos_with_docker(self):
+    def test_positive_create_with_repos_with_docker(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_contentviewfilter.py]: not sure about `with_docker` here

File: tests/foreman/api/test_discoveredhost.py [hunk 12]
```diff
@@ -227,11 +217,10 @@ def test_reboot_host(self):
         @Assert: Selected host should be rebooted successfully
 
         @Status: Manual
-
         """
 
     @stubbed()
-    def test_create_discovery_rule_1(self):
+    def test_positive_create_discovery_rule_with_simple_query(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_discoveredhost.py]: Maybe `test_positive_create_discovery_rule_with_ip`

File: tests/foreman/api/test_discoveredhost.py [hunk 13]
```diff
@@ -243,11 +232,10 @@ def test_create_discovery_rule_1(self):
         @Assert: Host should reboot and provision
 
         @Status: Manual
-
         """
 
     @stubbed()
-    def test_create_discovery_rule_2(self):
+    def test_positive_create_discovery_rule_with_complex_query(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_discoveredhost.py]: Maybe `test_positive_create_discovery_rule_with_cpu_count`. Let me know if I need to change that code on my side to yours approach

File: tests/foreman/api/test_docker.py [hunk 14]
```diff
@@ -1228,8 +1197,9 @@ def test_create_container_content_view(self):
                 )
                 self.assertEqual(container.tag, 'latest')
 
+    @tier2
     @run_only_on('sat')
-    def test_create_container_compute_resource_power(self):
+    def test_positive_create_compresource_power(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_docker.py]: I suggest to use `test_positive_power_compresource`

File: tests/foreman/api/test_docker.py [hunk 15]
```diff
@@ -1258,7 +1227,7 @@ def test_create_container_compute_resource_power(self):
 
     @tier2
     @run_only_on('sat')
-    def test_create_container_compute_resource_read_log(self):
+    def test_positive_create_compresource_read_log(self):
```
Comment (oshtaier) [inline tests/foreman/api/test_docker.py]: `test_positive_read_compresource_logs`

**Comment by oshtaier** [comment]
Please address comments in your next part refactoring

File: tests/foreman/api/test_discoveredhost.py (hunk 12 above)
Comment (oshtaier) [inline tests/foreman/api/test_discoveredhost.py]: btw, where did you lose tiers here? :)

**Pull request merged**